packages/react-dom/src/__tests__/ReactBrowserEventEmitter-test.internal.js

```diff
@@ -54,7 +40,6 @@ describe('ReactBrowserEventEmitter', () => {
     jest.resetModules();
     LISTENER.mockClear();
 
-    // TODO: can we express this test with only public API?
     EventPluginHub = require('events/EventPluginHub');
```

Comment:
I'm not sure if I was being clear enough.

Anything except `react` or `react-dom` themselves is considered private API. Nothing that starts with `events/` or has `/src/` in the path is public.